lib/Myriad/RPC/Implementation/Redis.pm

```diff
@@ -83,7 +83,7 @@ async method stop () {
 
 async method create_group ($rpc) {
     unless ($rpc->{group}) {
-        await $self->redis->create_group($rpc->{stream}, $self->group_name);
+        await $self->redis->create_group($rpc->{stream}, $self->group_name, '$', 1);
```

Comment:
actually we the create stream config enabled for the memory transports, that's why it wasn't caught by tests